frontend/io-webapp/src/session/Chat.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+const Chat = (props) => {
+  const sampleChat = [
+    {
+      id: 1,
+      username: "Alice",
+      message: "Hello",
+      time: "21:37",
+    },
+    {
+      id: 2,
+      username: "Bob",
+      message: "Hi",
+      time: "21:38",
+    },
+  ];
+
+  return (
+    <div className="chat">
+      <section>
+        <h3>Here be reactions</h3>
+        <form>
+          <input type="text" id="chatMessage" name="chatMessage" />
+          <button type="submit">Send</button>
+        </form>
+      </section>
+      {sampleChat.reverse().map((msg) => {
```

Comment:
Całkowicie zdaję sobie sprawę, że narazie to raczej mock, a nie docelowy widok, ale docelowo ładniej by było gdyby to `.reverse()` nie było wołane przy każdym rerenderze.

Reply:
To jest typowo mockowa rzecz, nie wiem jeszcze jak to będzie wyglądać przy socketach.